docs/00-prerequisite.md

```diff
@@ -14,9 +14,6 @@ git clone https://github.com/redhat-na-ssa/hobbyist-guide-to-rhoai.git
 
 # change into the repo directory
 cd hobbyist-guide-to-rhoai
-
-# checkout below branch
-git checkout hshishir-25ba6c
 ```
 
```

Comment:
can you add instructions for installing 'jq' and 'helm'

Reply:
Can we do this as a separate PR?

Reply:
@hemang-rh ^^

Reply:
yes we can address feedbacks in new PR